src/main/groovy/edu/oregonstate/mist/api/jsonapi/IOExceptionMapper.groovy

```diff
@@ -0,0 +1,31 @@
+package edu.oregonstate.mist.api.jsonapi
+
+import edu.oregonstate.mist.api.Error
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import javax.ws.rs.core.Response
+import javax.ws.rs.ext.ExceptionMapper
+import javax.ws.rs.ext.Provider
+
+/**
```

Comment:
Let's remove this default comment. It doesn't add any value that git log doesn't already provide.